src/Microsoft.Diagnostics.Runtime/Desktop/v45runtime.cs

```diff
@@ -550,6 +550,11 @@ internal override IRCWData GetRCWData(ulong rcw)
         }
         #endregion
 
+        public override IEnumerable<ClrException> GetSerializedExceptions()
```

Comment:
See the comment in LegacyRuntime.cs.  This should be overridden in DesktopBaseRuntime instead.